packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/GutenbergWebViewActivity.java

```diff
@@ -333,10 +333,18 @@ private void injectCssScript() {
                 String injectWPBarsCssScript = getFileContentFromAssets("gutenberg-web-single-block/wp-bar-override.css");
                 injectWPBarsCssScript = removeWhiteSpace(removeNewLines(injectWPBarsCssScript));
                 evaluateJavaScript(String.format(INJECT_CSS_SCRIPT_TEMPLATE, injectWPBarsCssScript));
+
+                String injectExternalCssScript = getOnGutenbergReadyExternalStyles();
```

Comment:
Thank you for testing and the review, @twstokes! 

> Do you think we should make the timing of both match to be consistent? For Android this would mean making the call at either onPageCommitVisible or onPageStarted, but we may want to summon an Android dev for the best advice. 😄

I followed the way [the existing `editor-style-overrides.css` file](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-bridge/common/gutenberg-web-single-block/editor-style-overrides.css) is loaded for both platforms as a pattern (i.e. under [this code block](https://github.com/WordPress/gutenberg/blob/0cda8b3e569faba16bc10aa39c60d44658e92268/packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/GutenbergWebViewActivity.java#L325-L342) on Android) under the assumption it's best for the new `external-style-overrides.css` file to load at the same time as that one. If we loaded one or the other at separate times, I worry that there could be cases where parts of the editor are hidden at differing times. WDYT?

Maybe there's a case for creating a separate issue to address the inconsistency for how UBE styles are loaded across platforms?

Reply:
That makes sense @SiobhyB - so, basically we're following patterns for their respective platforms. In that case that's all I had!